src/styles/button.scss

```diff
@@ -0,0 +1,34 @@
+.button {
+  &-buy {
+    margin-top: 16px;
+    color: $white;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background: $blue-accent;
+    border-style: hidden;
+    outline: auto;
+
+    text-align: center;
+    font-family: "Roboto", sans-serif;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    text-transform: uppercase;
+
+    &:hover {
```

Comment:
no need to duplicate all styles here
set only styles you want to change on hover